client/components/data/document-head/index.jsx

```diff
@@ -131,8 +95,6 @@ DocumentHead.propTypes = {
 export default connect(
 	state => ( {
 		formattedTitle: getDocumentHeadFormattedTitle( state ),
-		allLinks: getDocumentHeadLink( state ),
```

Comment:
These are the only usages of those 2 selectors. We could totally delete them as well!

Reply:
Err nope :smile: 

https://github.com/Automattic/wp-calypso/blob/0b770c9c7ade2d41684f9f78a5677cbd348a11a4/server/render/index.js#L177-L178

Reply:
We want these in our server-generated HTML, after all :wink: